magento/graphql-ce
# Added URL rewrites data to the product interface

### Description
Added URL rewrites data to the GraphQL product interface. So, now we have a possibility to fetch all URL rewrites for a product with request URL data and target URL parameters.

Also, the old `UrlRewrite` resolver has been renamed to `EntityUrl` to make the class name match its purpose. 

### Fixed Issues (if relevant)
1. https://github.com/magento/graphql-ce/issues/84

### Manual testing scenarios
- Create a product with name "configurable" and assign it to a category
- Use the following request to fetch the product data:

```
{
    products (search:"conf") {
        items {
            name,
            sku,
            description,
          	url_rewrites {
              url,
              parameters {
                name,
                value
              }
            }
        }
    }
}
```

- You should see 2 URL rewrites in the response like in the following example:

```
{
  "data": {
    "products": {
      "items": [
        {
          "name": "Configurable",
          "sku": "Configurable",
          "description": null,
          "url_rewrites": [
            {
              "url": "configurable.html",
              "parameters": [
                {
                  "name": "id",
                  "value": "5"
                }
              ]
            },
            {
              "url": "some-category/configurable.html",
              "parameters": [
                {
                  "name": "id",
                  "value": "5"
                },
                {
                  "name": "category",
                  "value": "3"
                }
              ]
            }
          ]
        }
      ]
    }
  }
}
```
